recipes/classix/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: classix
```

Comment:
I believe this name should be `ClassixClustering` to match the `pip install`.

Reply:
Thanks Joshua. In the beginning we hope this package can be  named as ``classix``, but in this name had been taken by others. So to install classix, we need to specify ``ClassixClustering``. But when you import ``classix``, we still use ``import classix``. We hope this can be consistent, so can we name it ``classix`` in conda forge?

Reply:
We changed it accordingly, thanks